bitnami/minio/Chart.yaml

```
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 6.1.13
+version: 6.1.14
```

```suggestion
version: 6.2.0
```
Since we are adding new features you should bump the minor version